csharp/BYOAuthFlow/Program.cs

```diff
@@ -0,0 +1,228 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using Google.Protobuf;
+using Google.Protobuf.WellKnownTypes;
+using Improbable.SpatialOS.Deployment.V1Alpha1;
+using Improbable.SpatialOS.Platform.Common;
+using Improbable.SpatialOS.PlayerAuth.V2Alpha1;
+using Improbable.Worker;
+using Improbable.Worker.Alpha;
+using Utils;
+using Deployment = Improbable.SpatialOS.Deployment.V1Alpha1.Deployment;
+using Locator = Improbable.Worker.Alpha.Locator;
+using LocatorParameters = Improbable.Worker.Alpha.LocatorParameters;
+
+namespace BYOAuthFlow
+{
+    internal class Program
+    {
+        /// <summary>
+        ///     PlEASE REPLACE ME.
+        ///     Your SpatialOS project name.
+        ///     It should be the same as the name specified in the local spaitalos.json file used to start spatiald.
```

Comment:
Typo: "spaitalos" 😂 😳 

Also, change `spatiald` to "the local API service" (we decided not to mention `spatiald` in the public docs).

Reply:
Just noticed these issues are present in the current live docs too - I'm happy to update them in a separate PR, or you can update them as part of this if that's easier.

Reply:
soudns good. Worth solving in a single PR.